java/rocksjni/sst_file_reader_iterator.cc

```diff
@@ -206,6 +206,28 @@ jint Java_org_rocksdb_SstFileReaderIterator_keyDirect0(
                                                   jtarget_off, jtarget_len);
 }
 
+/*
+ * This method supports fetching into indirect byte buffers;
+ * the Java wrapper extracts the byte[] and passes it here.
+ *
+ * Class:     org_rocksdb_SstFileReaderIterator
+ * Method:    keyByteArray0
+ * Signature: (J[BII)I
+ */
+jint Java_org_rocksdb_SstFileReaderIterator_keyByteArray0(
+    JNIEnv* env, jobject /*jobj*/, jlong handle, jbyteArray jkey, jint jkey_off,
+    jint jkey_len) {
+  auto* it = reinterpret_cast<ROCKSDB_NAMESPACE::Iterator*>(handle);
+  ROCKSDB_NAMESPACE::Slice key_slice = it->key();
+  jsize copy_size = std::min(static_cast<uint32_t>(key_slice.size()),
```

Comment:
define var actual_size=key_slice.size()

Reply:
I've done `auto slice_size = key_slice.size(); ` ? (actual suggests to me the presence of non-actual).